src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -134,13 +133,13 @@ private String getFormattedHostForGlobalUrl(EnvType env) {
     switch (env) {
       case DEV: {
         String devBoxOwner = this.env.split("-")[1];
-        return String.join(".", "www", devBoxOwner, envType.getWikiaDomain());
+        return String.join(".", "www", devBoxOwner, envType.getDomain());
       }
       case SANDBOX: {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested code block into a method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)